src/ttsnake.c

```diff
@@ -229,6 +234,14 @@ void draw (scene_t* scene, int number)
   if (number == GAME_OVER) { /* if displayed scene is the game over scene */
     /* we must show the final score */
     mvwprintw(main_window, NROWS*3/4, NCOLS/2-7, "Score: %d", block_count);
+    
+    /*Instructions to enter nickname*/
+    if(!entered_score) {
+      mvwprintw(main_window, NROWS*3/4 + 2, NCOLS/5-7, "Digite um nick de até 4 letras para salvar sua pontuacao.");
+      mvwprintw(main_window, NROWS*3/4 + 3, NCOLS/5-7, "Caso não deseje salvar, aperte '#'.");
+      mvwprintw(main_window, NROWS*3/4 + 4, NCOLS/5-7, "Caso deseje deletar um caractere, digite '&'.");
```

Comment:
```suggestion
      mvwprintw(main_window, NROWS*3/4 + 3, NCOLS/5-7, "In case you want to delete the last typed letter, type '&'.");

```

Reply:
I would translate this to "If you want to erase a character, type '&'."